storage/client/queries/queries.go

```diff
@@ -544,10 +550,10 @@ const (
 						TRUE
 					-- Special case to return are 'likely to be native transfers'.
 					WHEN $5::text = 'native_transfers' THEN
-						(txs.method = 'accounts.Transfer' OR (txs.method = 'evm.Call' AND (txs.body ->> 'data') = ''))
+						(txs.likely_native_transfer = TRUE)
```

Comment:
boolean_field = TRUE

is that our style?

Reply:
Fixed 🙈